baseline.gradle

```diff
@@ -36,7 +36,7 @@ subprojects {
     apply plugin: 'com.palantir.baseline-checkstyle'
     apply plugin: 'com.palantir.baseline-error-prone'
   }
-  apply plugin: 'com.palantir.baseline-scalastyle'
+  apply plugin: 'com.github.alisiikh.scalastyle'
```

Comment:
I haven't found any license either, but `com.palantir.baseline-scalastyle` is essentially using `org.github.ngbinh.scalastyle` (see https://github.com/palantir/gradle-baseline/blob/develop/gradle-baseline-java/build.gradle#L15) and `com.github.alisiikh.scalastyle` is a fork of it (I guess because `org.github.ngbinh.scalastyle` was updated in 2018 the last time)